### PR TITLE
dailymotion.com, comicbookrealm.com, allenetflixseries.nl, hollywoodpq.com

### PR DESCRIPTION
**Pull request by DandelionSprout**
Example pages:

```
! https://www.dailymotion.com/video/x46yl0a
dailymotion.com##div[class*="_adCell_"]
! https://comicbookrealm.com/series/12254/156387/tokyo-pop-mixx-my-little-pony-jr-cinemanga-soft-cover-3
comicbookrealm.com###rad
comicbookrealm.com###brad
comicbookrealm.com###sad
comicbookrealm.com###tad
! https://www.allenetflixseries.nl/clerks-the-animated-series/
allenetflixseries.nl##.textwidget > p:has(:scope > script[src*="/adsbygoogle"])
! https://hollywoodpq.com/lhilarante-webserie-mouvement-deluxe-avec-catherine-brunet-est-de-retour/
hollywoodpq.com##.oboxads-outstream
```

### CODE REVIEW

**Comment by jspenguin2017** [comment]
`https://www.dailymotion.com/video/x46yl0a`

**Comment by jspenguin2017** [comment]
`https://comicbookrealm.com/series/12254/156387/tokyo-pop-mixx-my-little-pony-jr-cinemanga-soft-cover-3`

**Comment by jspenguin2017** [comment]
`https://www.allenetflixseries.nl/clerks-the-animated-series/`

**Comment by jspenguin2017** [comment]
`https://hollywoodpq.com/lhilarante-webserie-mouvement-deluxe-avec-catherine-brunet-est-de-retour/`

File: dist/placeholder-buster.txt [hunk 1]
```diff
@@ -550,6 +550,15 @@ rule34.paheal.net##section:has(:scope > .blockbody > div[align="center"])
 
 # -------------------------------------------------------------------------------------------------------------------- #
 
+# French
+
+# https://github.com/NanoAdblockerLab/NanoContrib/pull/31
+hollywoodpq.com##.oboxads-outstream
```
Comment (smed79) [inline dist/placeholder-buster.txt:556]: https://hg.adblockplus.org/listefr/rev/f94c3f127172

**Comment by smed79** [comment]
hollywoodpq.com
https://hg.adblockplus.org/listefr/rev/f94c3f127172

dailymotion.com
https://github.com/easylist/easylist/pull/3645
https://hg.adblockplus.org/listefr/rev/f7267303ecf3

**Comment by jspenguin2017** [comment]
@smed79 OK, I subscribed to your PR, I'll clean up here once it's merged.

**Pull request merged**